src/bindings/python/src/openvino/runtime/opset14/ops.py

```diff
@@ -62,3 +65,103 @@ def inverse(
     }
 
     return _get_node_factory_opset14().create("Inverse", inputs, attributes)
+
+
+@nameable_op
+def max_pool(
+    data: NodeInput,
+    strides: List[int],
+    dilations: List[int],
+    pads_begin: List[int],
+    pads_end: List[int],
+    kernel_shape: TensorShape,
+    rounding_type: str = "floor",
+    auto_pad: Optional[str] = None,
+    index_element_type: Optional[str] = "i64",
```

Comment:
Can `ov.Type` be allowed as well? 

```suggestion
    index_element_type: Optional[Union[Type, str]] = "i64",
```

Reply:
I don't think so, we would have to have an additional logic in the operator wrapper, which would either translate `ov.Type` from Python to C++. It's currently being passed "raw" into `NodeFactory`.

https://github.com/openvinotoolkit/openvino/blob/902f1b1af0f5abff87c32dca62d261ce5d1d01c2/src/bindings/python/src/openvino/runtime/opset8/ops.py#L304-L318

If you wish we can consider adding that logic. cc @akuporos @jiwaszki do you recall if we do this for other ops?

Reply:
It works with `ov.Type`. Change applied.